frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1338,7 +1341,8 @@ const IR::Statement* ProgramStructure::convertPrimitive(const IR::Primitive* pri
 
 const IR::P4Action*
 ProgramStructure::convertAction(const IR::ActionFunction* action, cstring newName,
-                                const IR::Meter* meterToAccess) {
+                                const IR::Meter* meterToAccess,
+                                cstring counterToAccess) {
```

Comment:
This is a problem with the existing code for both direct_counter and direct_meter that should be addressed.